components/backup-stream/src/endpoint.rs

```diff
@@ -1049,6 +1036,29 @@ fn create_tokio_runtime(thread_count: usize, thread_name: &str) -> TokioResult<R
         .build()
 }
 
+pub enum BackupStreamResolver<RT, EK> {
+    // for raftstore-v1, we use LeadershipResolver to check leadership of a region.
+    V1(LeadershipResolver),
+    // for raftstore-v2, it has less regions. we use CDCHandler to check leadership of a region.
+    V2(RT, PhantomData<EK>),
```

Comment:
Why do we need `PhantomData<EK>`?

Reply:
because `CdcHandle<EK>` needs it, so we use PhantomData to make it compile